internal/printer/printer.go

```diff
@@ -512,6 +514,32 @@ func maybeConvertTransition(n *astro.Node) {
 			})
 		}
 	}
+	if transform.HasAttr(n, transform.TRANSITION_RELOAD) {
+		transitionReloadIndex := transform.AttrIndex(n, transform.TRANSITION_RELOAD)
+		attr := n.Attr[transitionReloadIndex]
+
+		if n.Type == astro.ElementNode &&
```

Comment:
could you add a comment here for which elements are allowed to have this directive?

Reply:
Done!